src/settings/MappingProfiles/ViewMappingProfile.js

```diff
@@ -253,22 +215,103 @@ export class ViewMappingProfile extends Component {
-          styles={styles}
-          record={record}
-          referenceTables={referenceTables}
-          injectedProps={injectedProps}
-        />
+        <AccordionSet>
+          <Accordion
+            id="summary"
+            label={<FormattedMessage id="ui-data-import.summary" />}
+          >
+            <ViewMetaData
+              metadata={mappingProfile.metadata}
+              systemId={SYSTEM_USER_ID}
+              systemUser={SYSTEM_USER_NAME}
+            />
+            <div data-test-name-field>
+              <KeyValue
+                value={name}
+                label={<FormattedMessage id="ui-data-import.name" />}
+              />
+            </div>
+            <KeyValue label={<FormattedMessage id="ui-data-import.incomingRecordType" />}>
+              <div data-test-incoming-record-type>
+                <FormattedMessage id={INCOMING_RECORD_TYPES[incomingRecordType].captionId} />
+              </div>
+            </KeyValue>
+            <KeyValue label={<FormattedMessage id="ui-data-import.folioRecordType" />}>
+              <div data-test-folio-record-type>
+                <FormattedMessage id={FOLIO_RECORD_TYPES[existingRecordType].captionId} />
+              </div>
+            </KeyValue>
+            <KeyValue label={<FormattedMessage id="ui-data-import.description" />}>
+              <div data-test-description>{mappingProfile.description || <NoValue />}</div>
+            </KeyValue>
+          </Accordion>
+          {tagsEnabled && (
+            <div data-test-tags-accordion>
+              <TagsAccordion
+                link={`data-import-profiles/mappingProfiles/${id}`}
+                getEntity={getEntity}
+                getEntityTags={getEntityTags}
+                renderForbidden={!tagsEnabled}
+              />
+            </div>
+          )}
+          <Accordion
+            id="mapping-profile-details"
+            label={<FormattedMessage id="ui-data-import.details" />}
+          >
+            {existingRecordType
+              ? (
+                <AccordionStatus>
+                  <Row between="xs">
+                    <Col>
+                      <MappedHeader
+                        mappedLabelId="ui-data-import.settings.profiles.select.mappingProfiles"
+                        mappedLabel="Field mapping"
+                        mappableLabelId={MAPPING_DETAILS_HEADLINE[existingRecordType]?.labelId}
+                        mappableLabel={MAPPING_DETAILS_HEADLINE[existingRecordType]?.label}
+                        headlineProps={{ margin: 'small' }}
+                      />
+                    </Col>
+                    <Col>
+                      <div data-test-expand-all-button>
+                        <ExpandAllButton />
+                      </div>
+                    </Col>
+                  </Row>
+                  {renderDetails[existingRecordType]}
+                </AccordionStatus>
+              )
+              : (<></>)
+            }
```

Comment:
Mmm.. I think you don't need a ternary operator here, I could be just 
```
{existingRecordType && (
  <AccordionStatus>
   ...
  </AccordionStatus>
)}
```
Try it. If you left an empty element for future implementation, please add a comment in code.

Reply:
Looks like some fly has bitten me, thanks for the comment!